src/EnergyPlus/OutputProcessor.cc

```diff
@@ -558,10 +444,12 @@ namespace OutputProcessor {
         // SUBROUTINE ARGUMENT DEFINITIONS:
 
         // SUBROUTINE PARAMETER DEFINITIONS:
-        static std::vector<std::string> const PossibleFreq({"deta", "time", "hour", "dail", "mont", "runp", "envi", "annu"});
+        static std::vector<std::string> const PossibleFreqs({"DETA", "TIME", "HOUR", "DAIL", "MONT", "RUNP", "ENVI", "ANNU"});
```

Comment:
Avoid consistently upper casing stuff.

Reply:
I know this is already merged, but ... these things can/should be `static constexpr std::array<std::string_view>` which are compiled into the executable rather than constructed (I understand that this is `static` and so only constructed once, but once is still more than zero times.